### ISSUE TITLE
[ECS-logging-*] Ensure attributes are used for linking

### ISSUE THREAD

**Issue by bmorelli25** [elastic/observability-docs#1393]
We should double-check the `elastic/ecs-logging-*` repos to ensure attributes are being used for links. An example of why this is necessary can be seen in https://github.com/elastic/ecs-logging-nodejs/pull/111.

- [x] ecs-logging-ruby (n/a)
- [x] ecs-logging-python (n/a)
- [x] ecs-logging-php (n/a)
- [ ] ecs-logging-nodejs (https://github.com/elastic/ecs-logging-nodejs/compare/main...colleenmcginnis:ecs-logging-nodejs:obs-docs-issue-1393)
- [x] ecs-logging-java (n/a)
- [ ] ecs-logging-go-logrus (https://github.com/elastic/ecs-logging-go-logrus/compare/main...colleenmcginnis:ecs-logging-go-logrus:obs-docs-issue-1393)
- [ ] ecs-logging-go-zap (https://github.com/elastic/ecs-logging-go-zap/compare/main...colleenmcginnis:ecs-logging-go-zap:obs-docs-issue-1393)
- [ ] ecs-logging-go-zerolog (https://github.com/elastic/ecs-logging-go-zerolog/compare/main...colleenmcginnis:ecs-logging-go-zerolog:obs-docs-issue-1393)
- [ ] ecs-logging (https://github.com/elastic/ecs-logging/compare/main...colleenmcginnis:ecs-logging:obs-docs-issue-1393)
- [ ] ecs-dotnet (https://github.com/elastic/ecs-dotnet/compare/main...colleenmcginnis:ecs-dotnet:obs-docs-issue-1393)

**Comment by colleenmcginnis** [comment]
I ran out of time before PTO because https://github.com/elastic/docs/pull/3150 needs to be merged first, but I linked the branches that are ready to go and I'll open PRs in the new year!

**Comment by colleenmcginnis** [comment]
I don't think this is relevant post-migration. Feel free to reopen if you disagree!
